### ISSUE TITLE
"High-Level" on first line

### ISSUE THREAD

**Issue by ZeroPointOneZero** [BSVino/JaiPrimer#29]
Maybe it's just me, but don't you think that that's extremely misleading to refer to a Systems Programming Language, as high-level? Since it's the first sentence everyone sees I think it makes a pretty big difference. It makes it appear that Jai is a scripting language or something.

I know you later say "Imperative" "C-like" and "Strongly Typed"
But still, none of those strongly imply that it's not a scripting language. You could say half of those about Objective C or Python.

It's not until we get to "Brief Description" do you clarify, and by then people might have skimmed the top and closed it by then.

I know it might not seem big but it just kinda bothers me since this might be the first thing new people see.

Might also be a cultural/generational gap difference in how the phrase "high-level" is interpreted since it's fairly relative.

**Comment by refi64** [comment]
I mean, technically it *is* high-level by popular definition. I mean, if C++ is considered high-level, then Jai definitely is.

**Comment by ErikWallstrom** [comment]
> "High-level language" refers to the higher level of abstraction from machine language.
https://en.wikipedia.org/wiki/High-level_programming_language

Everything above assembly is considered high level, basically

**Comment by LongBoolean** [comment]
Yeah it's just that now days we have built up higher levels, and to some, C++ is a lower level then the level they work in, so confusion from some is understandable. How big of an issue this is or whether or not we should care about that is up for debate I guess. Personally I don't think it is a huge issue. I would hope most people would check out the bullet points in the "Brief Description" section right below, before getting board and leaving the page.

The very next sentence says: 
>It is an imperative static/strongly typed C-style language, but with a variety of modern language features that C lacks.

@ZeroPointOneZero I kind of see your point, (I wouldn't say "extremely misleading", potentially confusing perhaps) personally I see the words "C-style" as a descriptor saying that the language can be used in places where C is currently used, therefore it needs to be fast, compiled, etc. (But I am reading this with non fresh eyes)  I could see someone reading "C-style" and confusing that to mean something related to syntax, which by that way of thinking, Python or JavaScript could potentially count as "C-style". 
If this is decided to be an issue, an easy fix would be to include the word "compiled" somewhere in that list of adjectives. (and while we are at it maybe separate those adjectives with some commas)
